fern/pages/trading/slippage.mdx

```diff
@@ -0,0 +1,87 @@
+---
+title: Slippage
+hide-feedback: true
+---
+
+Slippage refers to the difference between the expected price of a trade and the actual price at which the trade is executed. It occurs most commonly in markets with high volatility or low liquidity.
+
+---
+
+## Understanding Slippage  
+When you place an order on any exchange, the price you see before execution is not always the price you will get. The final execution price can change due to other trades occurring in the market before your order is fully processed. This price deviation is known as slippage.  
+
+Slippage can be positive or negative:  
+- **Positive Slippage**: You receive a better price than expected.  
+- **Negative Slippage**: You receive a worse price than expected.  
+
+---
+
+## Factors Influencing Slippage  
+1. **Market Volatility**  
+   In highly volatile markets, prices can change rapidly, increasing the likelihood of slippage.  
+
+2. **Liquidity Depth**  
+   Trades involving large volumes relative to the available liquidity in the order book are more prone to slippage.
+
+3. **Order Type**  
+   - **Market Orders**: Execute immediately but are more vulnerable to slippage, as they prioritize speed over price.  
+   - **Limit Orders**: Allow you to set a specific price, eliminating slippage risk but reducing the likelihood of immediate execution.
+
+---
+
+## Max Slippage  
+
+**Max slippage** defines the largest acceptable price deviation from the mark price. During execution, an order will be canceled if the price exceeds the slippage limit or falls outside the Market Price Band.
+
+---
+
+## Example of Slippage in Perpetual Futures  
+
+Let's say you want to go long on a perpetual futures contract for ETH/USDT:  
+
+### Scenario 1: Market Buy Order (Long)  
+- **Mark Price**: $3,500  
+
+You place a market buy order for 1 ETH.  
+- If your order executes at **$3,495**, this represents **positive slippage**, as you entered the trade at a better price.  
```

Comment:
Need to update this. With the $ it renders as if it's a mathex formula:
![image](https://github.com/user-attachments/assets/e3bb1ceb-cd39-44e6-924a-dc055ad051b5)

Reply:
Done